### DISCUSSION TITLE
Welcome to the Life Engine Discussions!

### DISCUSSION THREAD

**Discussion by MaxRobinsonTheGreat** [MaxRobinsonTheGreat/EvolutionSimulatorV2#3] [Announcements]
Please put feature requests, ideas, and recommendations here. Do not make issues for those, I will close them.

For bug reports, you may make issues.

**Comment by TheProcgenFreak** [comment]
I would LOVE if there were a setting where you could procedurally generate a creature with a seed. I love to see the nature of seed-based generation come hand to hand with cellular automata.

**Reply by TheProcgenFreak** [reply]
Sorry if that's too much.
